ee/modules/110-istio/templates/control-plane/iop.yaml

```diff
@@ -60,6 +60,12 @@ spec:
   meshConfig:
     rootNamespace: d8-{{ $.Chart.Name }}
     trustDomain: {{ $.Values.global.discovery.clusterDomain | quote }}
+    discoverySelectors:
+    - matchExpressions:
```

Comment:
If I get it right, it means all deckhouse and upmeter namespaces are excluded except istio, ingress-nginx, user-authn. Could you please leave a comment on why we need these three particular namespaces and why we exclude others (and also ones owned by upmeter)?

Reply:
Done.